sdk/storage/Azure.Storage.Files.Shares/tests/FileClientTests.cs

```diff
@@ -3161,6 +3193,24 @@ public async Task ChangeLeaseAsync_Error()
                 e => Assert.AreEqual("ResourceNotFound", e.ErrorCode));
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = ShareClientOptions.ServiceVersion.V2020_02_10)]
+        public async Task RenewLeaseAsync_Error()
```

Comment:
I didn't see a test for this exception, so I added it here before I forget to it.
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Files.Shares/src/ShareLeaseClient.cs#L1027
Let me know if there's already an existing test. I know we probably would have to remove this test if they eventually add in renew for Share Files.